src/pages/Mykyta/componets/CarDetail.tsx

```diff
@@ -357,13 +370,14 @@ const CarDetail: React.FC<CardProps> = ({
                       error={Boolean(errors.year)}
                       helperText={errors.year?.message}
                   />
+                  <FormControl sx={CardTextField}>
                   <Controller
                     name="body_type"
                     control={control}
                     defaultValue={data.body_type}
                     rules={{ required: true }}
                     render={({ field }) => (
-                      <Select {...field} sx={CardSelect}>
+                      <Select {...field} sx={CardSelect} MenuProps={menuProps}>
```

Comment:
You can just move value of `menuProps` to mui theme and override styles on theme level and then you don't need to pass this to every select in you code